estela-web/src/components/SpiderDetailPage/index.tsx

```diff
@@ -185,125 +598,21 @@ export class SpiderDetailPage extends Component<RouteComponentProps<RouteParams>
                                             </Col>
```

Comment:
If the buttons doesn't do anything make it disabled. If setting tabs doesn't make any functions you could disable it too